larray/core/session.py

```diff
@@ -401,10 +651,22 @@ def summary(self, template=None):
 
 
 def local_arrays(depth=0):
+    """
+    Returns a session containing all local arrays.
+    
+    Parameters
+    ----------
+    depth: int
+        depth of call frame to inspect. 0 is where local_arrays was called,
+        1 the caller of local_arrays, etc.
+
+    Returns
+    -------
+    Session
```

Comment:
We should probably mention (here) or above that the keys will be sorted alphabetically.